scss/civicrm/_mixins.scss

```diff
@@ -1,3 +1,11 @@
+/*
+* Use Open Sans as primary font.
+*/
+
+@mixin open_sans_font_family() {
```

Comment:
i don't know why you are using snake_case, while everywhere else we use the dasherized-style

Reply:
My bad, fixed it!